client/src/components/SharedComponents/TopBarComponents/Logo.jsx

```diff
@@ -1,26 +1,12 @@
 import React from 'react';
 import '../../../../dist/style.css';
```

Comment:
The import of the style.css file might not be needed here as it's linked directly on the index.html file we have which will apply the css to everything that gets rendered accordingly.